agent/tools.py

```
@@ -265,26 +256,12 @@ def stitch_patterns(self, pattern_names: list[str], initial_input: str) -> str:
         
         return current_input
 
-    async def get_overview(self) -> dict[str, Any]:
-        """
-        Provides an architecture summary, module map, and git health.
-        
-        Returns:
-            overview (dict[str, Any]): A dictionary containing architecture summary, module mapping, and git health information.
-        """
-        return await self.repowise.get_overview()
+    def get_overview(self) -> dict[str, Any]:
```

Ruff (0.15.12)

[error] Undefined name `Any`

(F821)